### PR TITLE
React edit-in-place component

### PR DESCRIPTION
**Pull request by washort**
This provides a react version of the `edit-in-place` component on query and dashboard pages.

### CODE REVIEW

**Comment by washort** [comment]
updated to 5.0.

File: client/app/components/groups/group-name.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ export default function init(ngModule) {
     transclude: true,
     template: `
       <h2 class="m-t-0">
-        <edit-in-place editable="$ctrl.canEdit()" done="$ctrl.saveName" ignore-blanks='true' value="$ctrl.group.name"></edit-in-place>&nbsp;
+        <edit-in-place class="edit-in-place" editable="$ctrl.canEdit()" onDone="$ctrl.saveName" ignore-blanks="'true'" value="$ctrl.group.name"></edit-in-place>&nbsp;
```
Comment (arikfr) [inline client/app/components/groups/group-name.js]: Why do we need to specify the class name?

`onDone` should probably be `on-done`, right?

**Comment by washort** [comment]
Fixed those issues.

**Comment by arikfr** [comment]
👍

**Pull request merged**